### PR TITLE
Contact form and formspree service integration

### PR DESCRIPTION
**Pull request by cuauh-cabrera**
Below are listed all the features included in this branch:

- [ ] Add a contact form.
- [ ] Add Formspree service integration.
- [ ] Add footer.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -184,7 +185,34 @@ <h2 class="about-card3__title">Skills</h2>
                 </article>
             </div>
         </section>
+        <section class="contact">
+            <div class="contact__headline">
+                <h2 class="contact__header">I'm always <br>interested in <br>hearing about new projects, so <br>if you'd like to <br>chat please
+                    get <br>in touch.</h2>
+            </div>
+            <form class="contact__form" action="https://formspree.io/f/mbjezwnw" method="post">
+                <input class="contact__form--input" name="name" type="text" id="name" maxlength="50" placeholder="Full name" required>
```
Comment (mukhammadrizooff) [inline index.html:194]: - [ ] [REQUIRED] Good job so far! `Name` input should have a `maxlength="30"` attribute., please give the correct value to `Name <input>` 🙏🏼

File: index.html [hunk 2]
```diff
@@ -184,7 +185,34 @@ <h2 class="about-card3__title">Skills</h2>
...
+                <input class="contact__form--input" name="email" type="email" id="email" maxlength="50" placeholder="Email address" required>
+                <textarea name="message" id="" cols="30" rows="10" maxlength="600" placeholder="Hey I'd like to connect to..." required></textarea>
```
Comment (mukhammadrizooff) [inline index.html:196]: - [ ] [REQUIRED] Good job so far! Message input should have a `maxlength="500"` attribute, please give the correct value to `<textarea>` 🙏🏼

File: index.html [hunk 3]
```diff
@@ -184,7 +185,34 @@ <h2 class="about-card3__title">Skills</h2>
...
+                <input class="action-btn" type="submit" value="Get in touch">
+            </form>
+            <h3>cabrera.cuau@gmail.com</h3>
+        </section>
     </main>
+    <footer class="footer">
+        <div class="footer__icons">
```
Comment (mukhammadrizooff) [inline index.html:203]: - [ ] [REQUIRED] Good job so far! Please fix the `footer` section design because they are placed wrong 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @TEAM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implmentations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (cuauh-cabrera) [reply index.html]: [SOLVED]

File: index.html (hunk 2 above)
Comment (cuauh-cabrera) [reply index.html]: [SOLVED]

File: index.html (hunk 3 above)
Comment (cuauh-cabrera) [reply index.html:204]: [SOLVED]

File: index.html [hunk 4]
```diff
@@ -184,7 +185,35 @@ <h2 class="about-card3__title">Skills</h2>
...
+            <ul>
+                <li><a href="https://twitter.com/cuau_cabrera"><img src="./assets/twitter.png"
+                            alt="This is a Twitter icon. Click on it to visit my Twitter profile"
+                            class="headline__icons--twitter"></a></li>
+                <li><a href="https://linkedin.com/in/cuaucabrera"><img src="./assets/Linkedin icon.png"
+                            alt="This is a Linkedin icon. Click on it to visit my Linkedin profile"
+                            class="headline__icons--linkedin"></a></li>
+                <li><a href="https://github.com/cuauh-cabrera"><img src="./assets/github.png"
+                            alt="This is a github icon. Click on it to visit my github profile"></a></li>
+            </ul>
```
Comment (huseyinbiyik) [inline index.html:214]: [OPTIONAL] Good job with fixing the order of the elements 👍  But please make sure you added enough space between the social media icons and mail address.
Screenshot 2023-04-05 at 8 09 17 PM
Screenshot 2023-04-05 at 8 09 26 PM

File: index.html [hunk 5]
```diff
@@ -184,7 +185,35 @@ <h2 class="about-card3__title">Skills</h2>
...
+        <section class="contact">
+            <div class="contact__headline">
+                <h2 class="contact__header">I'm always <br>interested in <br>hearing about new projects, so <br>if you'd like to <br>chat please
+                    get <br>in touch.</h2>
+            </div>
+            <form class="contact__form" action="https://formspree.io/f/mbjezwnw" method="post">
+                <input class="contact__form--input" name="name" type="text" id="name" maxlength="30" placeholder="Full name" required>
+                <input class="contact__form--input" name="email" type="email" id="email" maxlength="50" placeholder="Email address" required>
+                <textarea name="message" id="" cols="30" rows="10" maxlength="500" placeholder="Hey I'd like to connect to..." required></textarea>
+                <input class="action-btn" type="submit" value="Get in touch">
```
Comment (huseyinbiyik) [inline index.html:197]: [OPTIONAL] Unfortunately, the previous reviewer didn't notice that so I will keep it as optional. Please add the correct hover and active state for your submit button 👍 You can find the color value on design guideline on Figma.
Screenshot 2023-04-05 at 8 10 22 PM

**Review by huseyinbiyik — APPROVED** [review]
Hi team,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Form integrated with Formspree very well 👍 
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- PR description is descriptive 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions
-  @DebyGrey, please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML" 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by cuauh-cabrera** [comment]
@huseyinbiyik  Tank you so much. for your feedback! 👍

**Pull request merged**